### ISSUE TITLE
maybe use a slider, rather than a dropdown, for the date selector

### ISSUE THREAD

**Issue by lyricnz** [LukePrior/nbn-upgrade-map#294]
After https://github.com/LukePrior/nbn-upgrade-map/pull/293 is merged, maybe use a slider for the date-selector, like google earth does

**Comment by LukePrior** [comment]
I don't think this one is as vital given the time it takes to load in data but could be an interesting experiment to try and see what the experience is like.
